xml/ha_maintenance.xml

```diff
@@ -0,0 +1,375 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"
+ xml:id="cha.ha.maintenance">
+ <title>Executing Maintenance Tasks</title>
+ <info>
+  <abstract>
+   &maint-mode-basics;
+  </abstract>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:maintainer/>
+   <dm:status>editing</dm:status>
+   <dm:deadline/>
+   <dm:priority/>
+   <dm:translation>yes</dm:translation>
+   <dm:languages/>
+   <dm:release/>
+   <dm:repository/>
+  </dm:docmanager>
+ </info>
+ <sect1 xml:id="sec.ha.config.basics.maint.mode">
+  <title>Different Maintenance Modes</title>
+
+  <para>
+   With regard to that, the &hasi; provides maintenance modes and options on
+   several levels:
+  </para>
+
+  <variablelist>
+   <varlistentry>
+    <term>Cluster Maintenance Mode</term>
+    <listitem>
+     <para>
+      Use this option if you need to perform maintenance tasks and do not want
+      to receive any notification by the cluster during that time. In
+      maintenance mode, you can stop or restart cluster resources at
+      will&mdash;the &hasi; will not attempt to restart them. All resources
+      automatically become unmanaged: The &hasi; will cease monitoring them and
+      thus be oblivious to their status.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Maintenance Mode For Resources</term>
+    <listitem>
+     <para>
+      Use this option if you need to manually touch a service that is managed
+      by a resource (or if you need to adjust any components of this service).
+     </para>
+     <note>
+      <title>Monitoring Unmanaged Resources</title>
+      <para>
+       If you want to suspend management of an individual resource but still
+       want the cluster to monitor the resource and to report any failures,
+       adjust the <option>is-managed</option> meta attribute for this resource
+       instead of putting the resource into maintenance mode. See <xref
+       linkend="pro.ha.config.hawk.rsc.is-managed" xrefstyle="select:label"/> or <xref
+       linkend="sec.ha.manual_config.cli.maint.mode" xrefstyle="select:label"/>
+       for details.
+      </para>
+      <para>
+       For the difference between maintenance mode and
+       <option>is-managed</option>, see also
+       <xref linkend="tab.ha.basics.meta"/>.
+      </para>
+     </note>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Maintenance Mode for Nodes</term>
+    <listitem>
+     <para>
+      Use this option if you want to update software packages on a node that do
```

Comment:
and/or "do maintenance for plenty/all of  resources hosted on a node"?

Reply:
Changed, thanks.